ng-ncc-app/src/app/main/pages/rent/payment/payment.component.html

```diff
@@ -1,8 +1,21 @@
-<p class="govuk-inset-text">
-    Remind the caller that they can pay their rent via <a href="">https://www.hackney.gov.uk/rentaccount</a>
-    or the 24 hour payment line 020 8356 5050.<br>
-    Ask if they would like you to email or text them a link to either.
-</p>
+<div class="govuk-inset-text">
+    <p>Remind the caller that they can pay their rent via https://www.hackney.gov.uk/rentaccount or the 24 hour payment line 020 8356 5050.
```

Comment:
Perhaps there can be a text with a link to the site

Reply:
Good spot!